test/e2e/components/pill.js

```diff
@@ -0,0 +1,16 @@
+const { VlElement } = require('vl-ui-core').Test;
+const { By } = require('selenium-webdriver');
+
+class Pill extends VlElement {
```

Comment:
Er kon hier geen gebruik gemaakt worden van de vl-ui-pill e2e component?

Reply:
Nee, het gaat hier over de Webuniversum implementatie die net iets anders is dan de pill. Initieel had ik de VlPill gebruikt maar om dit werkende te krijgen waren er te ingrijpende changes nodig. Besproken met @pascalriquier en @coemans .

Reply:
Ja ik had zo een discussie opgevangen (eer)gisteren maar was niet meer zeker van de conclusie/redenen.